File: app/src/main/java/org/kindone/willingtodo/data/TaskListItem.java
package org.kindone.willingtodo.data;

import org.kindone.willingtodo.recyclerlist.RecyclerListItem;

/**
 * Created by kindone on 2015. 10. 16..
 */
public class TaskListItem implements RecyclerListItem{
    private Task task;

    public TaskListItem(Task task) {
        this.task = task;
    }

    public long getId() {

        return task.id;
    }
    public String getTitle() {
        return task.title;
    }

    public Task getTask() {

        return task;
    }
}

File: app/src/main/java/org/kindone/willingtodo/data/TaskListItem.kt
package org.kindone.willingtodo.data

import org.kindone.willingtodo.recyclerlist.RecyclerListItem

/**
 * Created by kindone on 2015. 10. 16..
 */
class TaskListItem(val task: Task) : RecyclerListItem {

    override fun getId(): Long {
        return task.id
    }

    override fun getTitle(): String {
        return task.title
    }
}
